src/Stratis.Bitcoin.Features.BlockStore/BlockStoreQueue.cs

```diff
@@ -212,7 +213,9 @@ private async Task RecoverStoreTipAsync()
             this.logger.LogTrace("()");
 
             var blockStoreResetList = new List<uint256>();
-            uint256 resetBlockHash = this.blockRepository.BlockHash;
+
+            // TODO ACTIVATION consider using height there
```

Comment:
I don't think we have blocks in repository indexed by height

Reply:
same as before. 
removed